qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -127,6 +127,29 @@ def _set(self, value, settings=None):
         )
 
 
+class DictionaryLanguageSetter(DefaultProfileSetter):
+
+    """Sets paths to dictionary files based on language codes."""
+
+    def __init__(self):
+        super().__init__('setSpellCheckLanguages', default=[])
+
+    def _set(self, value, settings=None):
+        if settings is not None:
+            raise ValueError("'settings' may not be set with "
+                             "DictionaryLanguageSetter!")
+        installed_langs = dict([(lang.code, lang.file)
+                                for lang in get_installed_languages()])
+        lang_files = []
+        for lang_code in value:
+            if lang_code in installed_langs:
+                lang_files.append(installed_langs[lang_code][:-5])
```

Comment:
Use `os.path.splitext(...)[0]` instead - however, maybe it'd be nicer for `installed_langs` to map from `lang.code` to `lang`, and make `Language` have some kind of `filename` attribute?